packages/components/music/UploadMusicModal/UploadTrack.tsx

```diff
@@ -150,9 +166,11 @@ export const UploadTrack: React.FC<Props> = ({ onUploadDone }) => {
     });
     if (!uploadedFiles.find((file) => file.url)) {
       console.error("upload file err : Fail to pin to IPFS");
-      setToastError({
+      setToast({
         title: "File upload failed",
         message: "Fail to pin to IPFS, please try to Publish again",
+        type: "error",
+        mode: "normal",
       });
       setIsUploadLoading(false);
```

Comment:
Just my suggestion :D  I see that we repeat several times the pattern: if (error) {console.error...setToast...setIsUploading(false)} ...

So you can write a try catch block:
```
try {
    if (error1) throw Error()
    if (error2) throw Error()
} catch {
   console.error
   setToast
   setIsUploading
}
```

Reply:
I never use `throw` to trigger a `catch` block, it's good.

I will not modify here because it's out of scope